images/entrypoint.sh

```diff
@@ -130,6 +132,9 @@ while [ "$1" != "" ]; do
         --multus-conf-file)
             MULTUS_CONF_FILE=$VALUE
             ;;
+        --multus-conf-dir)
```

Comment:
Let's also add a paragraph in the how-to-use doc in this section for the parameter: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#entrypoint-script-parameters

Reply:
Sure, will update the PR.